RecoTracker/IterativeTracking/python/InitialStepPreSplitting_cff.py

```diff
@@ -125,6 +125,37 @@
 )
 initialStepTrackCandidatesPreSplitting.MeasurementTrackerEvent = 'MeasurementTrackerEventPreSplitting'
 
+from Configuration.ProcessModifiers.trackingMkFitInitialStepPreSplitting_cff import trackingMkFitInitialStepPreSplitting
+from RecoTracker.MkFit.mkFitGeometryESProducer_cfi import mkFitGeometryESProducer
+import RecoTracker.MkFit.mkFitHitConverter_cfi as mkFitHitConverter_cfi
+import RecoTracker.MkFit.mkFitSeedConverter_cfi as mkFitSeedConverter_cfi
+import RecoTracker.MkFit.mkFitIterationConfigESProducer_cfi as mkFitIterationConfigESProducer_cfi
+import RecoTracker.MkFit.mkFitProducer_cfi as mkFitProducer_cfi
+import RecoTracker.MkFit.mkFitOutputConverter_cfi as mkFitOutputConverter_cfi
+# TODO: Split the MkHitConverter to two to avoid re-converting strip hits in initialStep
+mkFitHitsPreSplitting = mkFitHitConverter_cfi.mkFitHitConverter.clone(
+    pixelRecHits = 'siPixelRecHitsPreSplitting'
+)
```

Comment:
This made me think that it could be beneficial to look into splitting `MkFitHitConverter` for pixel and strip hits to avoid converting the strip hits twice.